src/sentry/utils/env.py

```diff
@@ -0,0 +1,5 @@
+import sys
+
+
+def in_test_environment():
+    return sys.argv[0] in {"pytest", "vscode"}
```

Comment:
This seems like a fine change but unrelated to the rest of the PR. Ideal would be to put this up as a separate PR for easier/atomic review.

Reply:
Ah sorry, I did a bad rebase where https://github.com/getsentry/sentry/pull/52775 got merged in!

Reply:
Removed in latest push.